### DISCUSSION TITLE
Timeout when using custom load

### DISCUSSION THREAD

**Discussion by SiqiKe** [NREL/REopt-API-Analysis#136] [8 - API: Application Programming Interface]
Hi there,

I am accessing your web tool in Python. I received an "Optimization exceeded timeout: 420 seconds." error when setting the load size to my customized list "load," a list with a len equal to 8760. (I tried rounding down each element to two decimal places to keep it simple, but it doesn't help). I can get a result if I don't use my custom load. Would you give some suggestions for fixing the problem?

The message is as follows:

'messages': {'info': 'When off_grid_flag is true, only PV, ElectricStorage, Generator technologies can be modeled.',
  'error': 'Optimization exceeded timeout: 420 seconds. Please visit https://github.com/NREL/REopt-API-Analysis/discussions for support with common API usage issues. You can also email reopt@nrel.gov with your run_uuid (2bfdd4aa-5c2a-4b16-addd-3cf3c4af2d89) for support.'

The following is my post:

post = {
    "Settings": {
        "off_grid_flag": True
    },       
    "Site": {
        "longitude": location.latitude,
        "latitude": location.longitude
    },
    "PV": {
        "array_type": 0,
        "installed_cost_per_kw": 2200,
        "om_cost_per_kw": 44
    },
    "Generator": {
        "installed_cost_per_kw": 500,
        "fuel_cost_per_gallon": 4.4
    },
    "ElectricLoad": {
        "loads_kw": load,
        #"doe_reference_name" : "LargeOffice",
        #"annual_kwh": 684623.0,
        #"year": 2017
    }
}

Best,

**Comment by adfarth** [comment]
Hello, are you still experiencing this issue? If so, you could try (in the web tool): 
- Increasing the "Solver optimality tolerance (%)"
- Increasing the "Optimization timeout (seconds)" (this is a new input in the web tool)
- Decreasing the "Load operating reserve requirement (%)" 
- Decreasing the "Minimum load met (%)"
